vnext/Microsoft.ReactNative/IReactPackageBuilder.idl

```diff
@@ -14,5 +14,6 @@ namespace Microsoft.ReactNative {
   interface IReactPackageBuilder {
     void AddModule(String moduleName, ReactModuleProvider moduleProvider);
     void AddViewManager(String viewManagerName, ReactViewManagerProvider viewManagerProvider);
+    void AddTurboModule(String moduleName, ReactModuleProvider moduleProvider);
```

Comment:
this is a breaking change. Apps depend on this interface.
Moreover this will break creating new apps as the template hasn't been updated to generate this new method.

Reply:
This interface is implemented by us. No apps are going to break. No new code is going to affected too. This is the power of the 'Builder' pattern. Applications must implement IReactPackageProvider with a single method that receives the IReactPackageBuilder as a parameter.

---
In reply to: [433524380](https://github.com/microsoft/react-native-windows/pull/5085#discussion_r433524380) [](ancestors = 433524380)

Reply:
What does `the template hasn't been updated` mean? In my understanding I have already provided implementation to this new function. Is there any other place out side of Microsoft.ReactNative that also implements this interface?